src/domain/chainContext.ts

```diff
@@ -97,16 +97,18 @@ export class ChainContext {
           currentBlockNumber = await provider.getBlockNumber();
           toBlock = toBlock > currentBlockNumber ? currentBlockNumber : toBlock;
 
-          _(
+          log.trace(
```

Comment:
THANKSSSSSSSSSSSSSS, so nice to see `_` gone

Reply:
![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia1.tenor.com%2Fimages%2F5c52dced74b4628ca26bfb2582d03d25%2Ftenor.gif%3Fitemid%3D7425816&f=1&nofb=1&ipt=acb343213eb06257f74bc9b7e85696168ebc1de01945ffbf4e31405a6293ebbd&ipo=images)

Reply:
You're welcome. `_` is retired on a beach drinking cocktails.